datumaro/plugins/coco_format/extractor.py

```diff
@@ -100,6 +107,22 @@ def _load_person_kp_categories(self, loader):
 
         return categories
 
+    @staticmethod
+    def _load_panoptic_config(path):
+        with open(path, 'r') as f:
+            import json
```

Comment:
This import can be moved to the top of the file

Reply:
Fixed